src/components/NewTab/style.scss

```diff
@@ -1,3 +1,9 @@
+body {
+    background-image: url(https://framedsc.com/HOFWallpaper/static/media/Wallpaper-1.35bf202a7c977cc76aef.png);
```

Comment:
I removed it, i'm not sure why you put the background here
It's also showing in the other pages and i'm not a big fan